saas/api/metrics.py

```diff
@@ -157,10 +176,13 @@ def get(self, request, *args, **kwargs): #pylint: disable=unused-argument
 class RevenueMetricAPIView(DateRangeContextMixin, ProviderMixin,
                            GenericAPIView):
     """
-    Retrieves 12-month trailing revenue
+    Retrieves 12-month trailing revenue.
```

Comment:
Not period ('.') - pun intended at the end of the title. To generate consistent [API documentation](https://www.djaodjin.com/docs/reference/djaoapp/latest/api/#billing).